media-sound/beets/beets-2.0.0.ebuild

```diff
@@ -0,0 +1,174 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} )
+PYTHON_REQ_USE="sqlite"
+
+# These envvars are used to treat github tarball builds differently
+# from pypi sources. Enable where required
+: ${IS_VCS_SOURCE="no"}
+: ${UPDATE_VERSION="no"}
+
+inherit distutils-r1 bash-completion-r1 multiprocessing optfeature
+
+if [[ ${PV} == "9999" ]]; then
+	EGIT_REPO_URI="https://github.com/beetbox/beets.git"
+	inherit git-r3
+else
+	MY_COMMIT=c75f07a0da6c622d3cd0f5aad0a08edaea360dad
```

Comment:
This commit doesn't correspond to the 2.0.0 tag. Fix it to just download the tag (or if you really want, check PV for *_p* before doing the commit thing.)

Reply:
oh my, silly of me to not notice this.. will fix momentarily